discord/abc.py

```diff
@@ -1350,6 +1351,7 @@ async def send(
         view: View = ...,
         suppress_embeds: bool = ...,
         silent: bool = ...,
+        poll: Poll = ...,
```

Comment:
This doesn't seem right?
Unless something changed, you can't send a poll with any other form of "content", so there should be no overloads which accept both Poll and any other kind (e.g. content, view, embeds, attachments)

Reply:
I've been testing polls and they do can be sent with embeds, views and contents.
![thingy](https://github.com/Rapptz/discord.py/assets/108473820/d5d4eaba-f85c-4426-a4ed-14eec95799fe)